lightning/config/config.go

```diff
@@ -337,7 +337,7 @@ func NewConfig() *Config {
 			BuildStatsConcurrency:      20,
 			DistSQLScanConcurrency:     100,
```

Comment:
`DistSQLScanConcurrency ` will decide the real concurrency  how many requests to each region are executed concurrently.

Reply:
It may not exceed the total thread number, which is sum of the size of coprocessor pool of each TiKV instance of the whole Cluster,